src/Amalgam/KnnCache.h

```diff
@@ -67,13 +68,11 @@ class KnnNonZeroDistanceQuerySBFCache
 					}
 				}
 
-				Concurrency::threadPool.CountCurrentThreadAsPaused();
```

Comment:
This method 'CountCurrentThreadAsResumed()' are no longer used anywhere but they are still in ThreadPool.h. Should they be removed?